frontend/core/blocks/image-augmentations/computations.py

```diff
@@ -0,0 +1,354 @@
+        test_name = "random_grid_shuffle"
+        description = ''
+        if parameter < 0:
+            parameter = 1
+        transform = A.Compose([A.RandomGridShuffle(grid=[int(round(parameter)), int(round(parameter))], always_apply=True)])
+        transformed = transform(image=input_image_numpy)
+        transformed_image = transformed["image"]
+        return {'test_name': test_name, 'parameters': parameter, 'image': transformed_image, 'description': description}
+
+    def gaussian_noise(input_image_numpy, parameter=250):
+        test_name = "gaussian_noise"
+        description = ''
+        transform = A.Compose([A.GaussNoise(var_limit=[int(round(parameter)), int(round(parameter))], always_apply=True)])
+        transformed = transform(image=input_image_numpy)
+        transformed_image = transformed["image"]
+        return {'test_name': test_name, 'parameters': parameter, 'image': transformed_image, 'description': description}
+
+    def round_to_odd(x: float) -> int:
+        """Rounds a float number to the closest odd integer."""
+        rounded = round(x)
+        if rounded % 2 == 0:
+            return rounded - 1 if x - rounded < 0 else rounded + 1
+        else:
+            return rounded
+
+    def motion_blur(input_image_numpy, parameter=10):
+        test_name = "motion_blur"
+        description = ''
+        if parameter < 3:
+            parameter = 3
+        parameter = round_to_odd(parameter)
+        transform = A.Compose([A.MotionBlur(blur_limit=[int(round(parameter)), int(round(parameter))], always_apply=True)])
+        transformed = transform(image=input_image_numpy)
+        transformed_image = transformed["image"]
+        return {'test_name': test_name, 'parameters': parameter, 'image': transformed_image, 'description': description}
+
+    def horizontal_flip(input_image_numpy, parameter=1):
+        test_name = "horizontal_flip"
+        description = ''
+        parameter = 1
+        transform = A.Compose([A.HorizontalFlip(always_apply=True)])
+        transformed = transform(image=input_image_numpy)
+        transformed_image = transformed["image"]
+        return {'test_name': test_name, 'parameters': parameter, 'image': transformed_image, 'description': description}
+
+    def vertical_flip(input_image_numpy, parameter=1):
+        test_name = "vertical_flip"
+        description = ''
+        parameter = 1
+        transform = A.Compose([A.VerticalFlip(always_apply=True)])
+        transformed = transform(image=input_image_numpy)
+        transformed_image = transformed["image"]
+        return {'test_name': test_name, 'parameters': parameter, 'image': transformed_image, 'description': description}
+
+    def horizontal_vertical_flip(input_image_numpy, parameter=1):
+        test_name = "vertical_flip"
+        description = ''
+        parameter = 1
+        transform = A.Compose([
+            A.HorizontalFlip(always_apply=True),
+            A.VerticalFlip(always_apply=True)])
```

Comment:
is there some other parameters for transformations that can be used it so then it limits the use of your function

Reply:
All the parameters should be hardcoded, otherwise, we need to tell the user what they can put as a value for each parameter for each transformation, which is hard to explain, the goal is to apply all the transformations without worrying about parameters.